### PR TITLE
Added docs for bg health

### PR DESCRIPTION
**Pull request by ThomasJRyan**

### CODE REVIEW

File: docs/health/health_background/background.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      - Specify background device connection
+      - **--health-bg-devices uut uut2 uut3**
+    * - --health-bg-checks
+      - Specify checks to run
+      - **--health-bg-checks cpu memory core logging**
+    * - --health-bg-config
+      - Specify pyATS Health Check configuration yaml file
+      - **--health-bg-config health_config.yaml**
+    * - --health-bg-connection
+      - Specify pyATS Health Check testbed connection (Cannot be telnet)
```
Comment (tahigash) [inline docs/health/health_background/background.rst:67]: I think what you meant was console connection. if telnet is not for console, you can have multiple connection even over telnet.
```suggestion
      - Specify pyATS Health Check testbed connection (Cannot be console connection)
```

File: docs/health/health_background/background.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      - **--health-bg-connection ssh**
+    * - --health-bg-v
+      - Increase output verbosity
```
Comment (tahigash) [inline docs/health/health_background/background.rst:70]: elaborate a bit more?

File: docs/health/health_background/background.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    pyats run job job.py --health-bg-check cpu memory --health-bg-connection ssh
+
+Run bg health on only uut::
+
+    pyats run job job.py --health-bg-check cpu memory --health-bg-devices uut
+
+Supply a custom health.yaml file::
+
+    pyats run job job.py --health-bg-check cpu memory --health-bg-file health.yaml
```
Comment (tahigash) [inline docs/health/health_background/background.rst:95]: I like these simple examples!

**Review by tahigash — APPROVED** [review]
approved with comments

File: docs/health/health_background/background.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
+.. _Background-health-check:
+
+Background Health Check
```
[the remark this answers is not in the document]
Comment (jeaubin) [reply docs/health/health_background/background.rst:3]: pyATS Health Check Background

File: docs/health/index.rst [hunk 5]
```diff
@@ -30,7 +30,7 @@ Using pyATS Health Check is very easy. Just list the above heath check names by
 
     pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core
 
-If you want to check only cpu, please specify `cpu` to `--health-checks`. **The pyATS Health Check specified by `--health-checks` runs as a post-processor after each test case.**
+If you want to check only cpu, please specify `cpu` to `--health-checks`. **The pyATS Health Check specified by `--health-checks` runs as a post-processor after each test case.** If you would like to continuously gather health information throughout the entire run concurrently, check out :ref:`pyATS Background Health Check <Background-health-check>`.
```
Comment (LukasMcClelland) [inline docs/health/index.rst:33]:
```suggestion
If you want to check only cpu, please specify `cpu` to `--health-checks`. **The pyATS Health Check specified by `--health-checks` runs as a post-processor after each test case.** If you would like to continuously gather health information throughout the entire run concurrently, check out :ref:`pyATS Health Check Background<Background-health-check>`.
```

**Review by LukasMcClelland — APPROVED** [review]
Swanky

**Pull request merged**